pages/recipes/supabase.mdx

```diff
@@ -1,24 +1,157 @@
 import { Callout } from "nextra/components";
 
-# Supabase
+# Overview
+
+This page is broken down into two sections:
+
+- [Seeding a supabase local development stack](/recipes/supabase#seeding-a-supabase-local-development-stack)
+- [Restoring to a supabase local development stack](/recipes/supabase#restoring-to-a-supabase-local-development-stack)
+
+![I absolutely love Supabase!](/recipes/supabase/snappy-holding-supabase-logo.svg)
+
+## Seeding a supabase local development stack.
+
+Supabase makes it easy to set up a [local environment](https://supabase.com/docs/guides/cli/local-development) linked to your Supabase project. Snaplet can extend this functionality by helping you seed your local database. This ensures that you have readily available data to work with initially and for any future feature or testing needs.
+
+In this short guide, we will use the [nextjs + supabase template](https://vercel.com/templates/next.js/supabase), to demonstrate how one could use the `snaplet generate` command to seed their local supabase stack.
+
+### Prerequisites
+
+- Supabase account
+- [Supabase CLI installed](https://supabase.com/docs/guides/cli/getting-started)
+- [Snaplet CLI installed](https://docs.snaplet.dev/getting-started/start-here#getting-started-with-snaplet-cli)
+
+#### 1. Create a Supabase project
+
+Create a new project in the Supabase Dashboard by visiting https://supabase.com/
+
+![Creating a supabase project](/recipes/supabase/supabase_new_project.webp)
+
+#### 2. Clone the nextjs + supabase template
+
+2.1. Create a Next.js app using the supabase starter template by running the `npx` command:
+
+```bash
+npx create-next-app -e with-supabase
+```
+
+**2.2.** Use `cd` to change into the newly created directory:
+
+```bash
+cd name-of-new-app
+```
+
+#### 3. Apply your migrations to the Supabase project.
+
+**3.1.** Link your nextjs starter template to your supabase project
+
+```bash
+supabase init
+```
+
+**3.2.** Apply the migrations in `/supabase/migrations` to your project
+
+> Your target database must have a structure, for the snaplet generate command to work.
```

Comment:
Do we need to provide additional context for this? Is there a resource we can point to for this, or is it intuitive and idiomatic for most developers to understand what we're asking them @khaya-zulu ?

Reply:
Good point, I'm trying to explain to the user why they need to apply the migrations in this step (like "a just so you know"). I think I will make more it clearer